include/swift-c/DependencyScan/DependencyScan.h

```diff
@@ -507,6 +507,68 @@ swiftscan_diagnostic_get_source_location(swiftscan_diagnostic_info_t diagnostic)
 SWIFTSCAN_PUBLIC void
 swiftscan_diagnostics_set_dispose(swiftscan_diagnostic_set_t* diagnostics);
 
+//=== Scanner File System Cache Diagnostics -------------------------------===//
+
+///  The kind of scanning file system cache out-of-date entries.
+typedef enum {
+  /// The entry is negatively stat cached (which indicates the file did not
+  /// exist the first time it was looked up during scanning), but the cached
+  /// file
+  /// exists on the underlying file system.
+  NegativelyCached,
```

Comment:
The current convention for other `enum`s in this file seems to be all caps snake case with a `SWIFTSCAN_` prefix.

Reply:
Nit: It should also be moved higher up in the header to be under:
```
//=== Public Scanner Data Types
```